sample-authz/src/main/java/net/trajano/ms/example/authz/HazelcastConfiguration.java

```diff
@@ -22,18 +36,37 @@
     @Bean
     public Config hazelcastConfig() {
 
-        return new Config()
+        final EntryListenerConfig listener = new EntryListenerConfig();
+        if (LOG.isDebugEnabled()) {
+            listener.setLocal(true)
+                .setImplementation(entryListener);
+        }
+        final Config config = new Config()
             .setInstanceName(instanceName)
             .setProperty("hazelcast.logging.type", "slf4j")
             .addMapConfig(new MapConfig()
                 .setName(TokenCache.ACCESS_TOKEN_TO_CLAIMS)
-                .setTimeToLiveSeconds(accessTokenExpirationInSeconds))
+                .setTimeToLiveSeconds(accessTokenExpirationInSeconds)
+                .setMaxIdleSeconds(accessTokenExpirationInSeconds)
+                .addEntryListenerConfig(listener))
             .addMapConfig(new MapConfig()
                 .setName(TokenCache.REFRESH_TOKEN_TO_ACCESS_TOKEN)
-                .setTimeToLiveSeconds(refreshTokenExpirationInSeconds))
+                .setTimeToLiveSeconds(refreshTokenExpirationInSeconds)
+                .setMaxIdleSeconds(refreshTokenExpirationInSeconds)
+                .addEntryListenerConfig(listener))
             .addMapConfig(new MapConfig()
                 .setName(TokenCache.REFRESH_TOKEN_TO_CLAIMS)
-                .setTimeToLiveSeconds(refreshTokenExpirationInSeconds));
+                .setTimeToLiveSeconds(refreshTokenExpirationInSeconds)
+                .setMaxIdleSeconds(refreshTokenExpirationInSeconds)
+                .addEntryListenerConfig(listener));
+
+        //      config.setListenerConfigs(Arrays.asList(new ListenerConfig()
+        //      .setClassName(HazelcastListener.class.getName())));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ACommentedOutCodeLine)